2021-02-03 09:43:29.639733 -- <module> --- (pymysql.err.OperationalError) (2006, "MySQL server has gone away (ConnectionAbortedError(10053, '你的主机中的软件中止了一个已建立的连接。', None, 10053, None))") [SQL: 'SELECT `TagDetail`.`ID` AS `TagDetail_ID`, `TagDetail`.`AreaName` AS `TagDetail_AreaName`, `TagDetail`.`DeviceNum` AS `TagDetail_DeviceNum`, `TagDetail`.`IP` AS `TagDetail_IP`, `TagDetail`.`Port` AS `TagDetail_Port`, `TagDetail`.`COMNum` AS `TagDetail_COMNum`, `TagDetail`.`FEFportIP` AS `TagDetail_FEFportIP`, `TagDetail`.`Area` AS `TagDetail_Area`, `TagDetail`.`Description` AS `TagDetail_Description`, `TagDetail`.`EnergyClass` AS `TagDetail_EnergyClass`, `TagDetail`.`Address` AS `TagDetail_Address`, `TagDetail`.`Type` AS `TagDetail_Type`, `TagDetail`.`Equipment` AS `TagDetail_Equipment` \nFROM `TagDetail`']

2021-02-03 09:44:49.725324 -- <module> --- (pymysql.err.InternalError) (1054, "Unknown column 'EquipmnetID' in 'field list'") [SQL: 'INSERT INTO `IncrementElectricTable` (`IncremenValue`, `IncremenType`, `Unit`, `EquipmnetID`, `Address`, `CollectionDate`, `AreaName`) VALUES (%(IncremenValue)s, %(IncremenType)s, %(Unit)s, %(EquipmnetID)s, %(Address)s, %(CollectionDate)s, %(AreaName)s)'] [parameters: {'IncremenValue': '64.0', 'IncremenType': '电', 'Unit': 'KW/h', 'EquipmnetID': None, 'Address': 'COM2.KT9F.总有功电量', 'CollectionDate': '2021-02-03 09:44:49', 'AreaName': '空调智能电表9F'}]

2021-03-03 14:26:02.350593 -- <module> --- not all arguments converted during string formatting

2021-03-24 14:59:05.378934 -- <module> --- not all arguments converted during string formatting

